relay-server/src/actors/project_buffer.rs

```diff
@@ -198,7 +204,12 @@ impl BufferService {
         let db = SqlitePoolOptions::new().connect_with(options).await?;
         sqlx::migrate!("../migrations").run(&db).await?;
 
-        Ok(())
+        let count: i64 = sqlx::query("SELECT count(*) FROM envelopes")
```

Comment:
We could also get the number of bytes here, and create a metric for that, but it's a heavy operation when the db is big:

```suggestion
        let count: i64 = sqlx::query("SELECT count(*), sum(length(envelope)) FROM envelopes")
```